hasher-matcher-actioner/hmalib/lambdas/aws_secrets.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import boto3
+import base64
+import typing as t
+import os
+import ast
+from botocore.exceptions import ClientError
+
+from threatexchange.api import ThreatExchangeAPI
+
+AWS_REGION = os.environ["AWS_REGION"]
```

Comment:
(not sure): I suspect that boto actually has a method to get this loaded by its own internal mechanism, which might be more flexible than the environment.

my_session = boto3.session.Session()
my_region = my_session.region_name

Reply:
Good to know!